src/modules/common/components/admin-only.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+const AdminOnly = ({ children, user }) =>
+  <div>
+    {user && user.admin ? children : null}
```

Comment:
Instead of the `AdminOnly` component, let's just pass the user prop into the `SiteFooter` and check if the user is an admin there and if so, render the `AdminToggle` component.